eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/push/AsyncHTTPPushRequest.java

```diff
@@ -328,7 +325,7 @@ ClientRetCode processResponseContent(String content) {
             Map<String, Object> ret =
                 JsonUtils.parseTypeReferenceObject(content, new TypeReference<Map<String, Object>>() {
                 });
-            Integer retCode = (Integer) ret.get("retCode");
+            Integer retCode = (Integer) Objects.requireNonNull(ret).get("retCode");
```

Comment:
How about replace "retCode" with ProtocolKey.RETCODE

Reply:
@mxsm I have updated the required changes. Please review.